### DISCUSSION TITLE
Failed to invite user: failed to make invite request: null value in column "email" of relation "profiles" violates not-null constraint

### DISCUSSION THREAD

**Discussion by organicnz** [supabase/supabase#12486] [Questions]
Hi folks,

I'm encountering a strange error `Failed to invite user: failed to make invite request: null value in column "email" of relation "profiles" violates not-null constraint`. Not sure if it is a bug considering that the email column doesn't contain any null value :)

Thank you :)

**Comment by GaryAustin1** [comment]
This seems to be an issue with a profiles table if you have it.   Maybe an insert trigger on auth.users not populating the email column on 
 profile table?

**Reply by organicnz** [reply]
Yeah, the email column was uncommented, and when I commented it out in the `handle_new_user` function, the error disappeared. However it pops another one, and I'm not sure how to handle it within the `handle_new_user` function :))

Error: 
`insert or update on table "profiles" violates foreign key constraint "profiles_user_address_new_fkey"`

Screenshots:
![image](https://user-images.githubusercontent.com/4765090/219580210-2c22467f-0190-457b-9799-eeec78b11132.png)

![image](https://user-images.githubusercontent.com/4765090/219580247-594e0e31-3eb8-4f65-8437-f6d4a7fcc378.png)

![image](https://user-images.githubusercontent.com/4765090/219586744-8ba96660-2762-4a9a-90db-97a30b5d680a.png)

![image](https://user-images.githubusercontent.com/4765090/219608290-a458f699-9df5-4de9-8336-fb8450947bb5.png)

**Reply by organicnz** [reply]
Hm, just curious maybe it's not a good practice to use the Foreign Key Relation `profile_id -> public.profiles.id` as the Primary Key in `address` table :)
